app/js/lib/foxbox/sequential-timer.js

```diff
@@ -0,0 +1,94 @@
+const p = Object.freeze({
```

Comment:
We start to have a lot of different files in `app/js/foxbox` and I think we need some sort of organisation.
Maybe we can start by placing all the base classes that other classes are extending in a subfolder (`base` or `extends`...). What do you think?